content/topics/github/intro/_index.md

```diff
@@ -19,29 +19,27 @@ Please go through this content and do your best to understand all the things.
 
 ## Github versus Git
 
-One thing to know is that Git and Github are not the same thing. Git is a foundational tool, you don't need to use Github if you want to use Git.
+One thing to know is that Git and Github are not the same things. Git is a foundational tool; you don't need to use Github if you want to use Git.
 
-Github is a web app and a collection of tools that make use of Git.
+Github is a web application and a collection of tools that make Git useful.
 
 Github has a few competitors. For example [GitLab](https://gitlab.com/) and [BitBucket](https://bitbucket.org/) are both quite awesome. You can even host your own github alternative on your own server one day, [Gogs](https://gogs.io/) is an example of that.
 
 ## IMPORTANT: Course expectations
 
-During your course here with us, you'll be expected to use github like a professional developer. Professional developers use the "command line". The documentation we gave you here should help you get to grips with the basics of what git and github are about. But we don't really do things that way.
+During your course here with us, you'll be expected to use GitHub like a professional developer. Professional developers use the "command line". The documentation we give you here should help you get to grips with the basics of what git and GitHub are about. But we don't do things that way.
 
 You'll learn more about how to use the command line in this project:
 {{% contentlink path="projects/git-exercises" %}}. We'll cover that later, but if you feel like doing that now, go wild :)
```

Comment:
on the actual site, this link works so I'm not sure if it's linked correctly here on the file.